src/components/tutorials-landing-view/certification-content-card-link/index.tsx

```diff
@@ -33,11 +28,19 @@ const CertificationContentCardLink = ({
 		CERTIFICATION_PROGRAM_SLUGS_TO_BACKGROUND_IMAGES[certification.slug]
 	const title = certification.title
 	const description = certification.description
-	const href = `/certifications/${certification.slug}`
+	/**
+	 * Special case for the consul certification to link to it's specfic header
+	 * {@link https://developer.hashicorp.com/certifications/security-automation#consul-associate-(003)-details}
```

Comment:
This is great! Should we also do the anchor in the other places where folks are clicking a link and explicitly expecting to get to the consul exam info? Like here:
![image](https://github.com/hashicorp/dev-portal/assets/47227843/c0d0b49b-a741-4094-8df1-6e288b42ce25)

Reply:
Nice callout, I'll open an Asana ticket, this PR is already getting pretty cluttered